features/issues/components/issue-list/issue-row.tsx

```diff
@@ -56,7 +56,6 @@ export function IssueRow({ projectLanguage, issue }: IssueRowProps) {
   return (
     <Row>
       <IssueCell>
-        <F.Checkbox size={F.CheckboxSize.md} state={F.CheckboxState.checked} />
```

Comment:
Might refer to this [article](https://blog.logrocket.com/building-custom-checkbox-react/)